deploy/examples/external/create-external-cluster-resources-tests.py

```diff
@@ -0,0 +1,287 @@
+"""
```

Comment:
Let's find a way to avoid duplicating the python scripts. Is there any difference from the original? If not, we can add a symlink for it to appear in this directory.

Reply:
I have tried to do the symlink. locally it works and after pushing the PR it is getting the pylint error and cannot find the file.  
I am using the following command: 
```
ln -s rook/deploy/examples/create-external-cluster-resources.py rook/deploy/examp
les/external
ln -s rook/deploy/examples/create-external-cluster-resources-tests.py rook/deploy
/examples/external
```

Reply:
pylint error is resolve by creating the symlink from the external directory.